src/components/index.ts

```diff
@@ -45,3 +45,6 @@ export type { BreadcrumbsProps } from "./Breadcrumbs/Breadcrumbs"
 
 export { default as ThemeWrapper } from "../theme/ThemeWrapper"
 export type { IThemeWrapperProps as ThemeWrapperProps } from "../theme/ThemeWrapper"
+
+export { default as Uploader } from "./Uploader/Uploader"
+export type { IUploaderProps } from "./Uploader/Uploader"
```

Comment:
```suggestion
export type { IUploaderProps as UploaderProps } from "./Uploader/Uploader"
```
Just to follow the naming pattern of the other exported interfaces of `{Component}Props`